ios/update_project.rb

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/ruby
+require 'xcodeproj'
+
+# This should be the Xcode ID for the team
+dev_team = "MZ11Z1Z11Z";
+
+project_path = "iOS/Electron-Cash.xcodeproj";
+
+# Create project object
+project = Xcodeproj::Project.open(project_path);
+
+lib = '/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS.sdk/usr/lib/libxml2.tbd'
```

Comment:
I wonder if you can use `xcode-select -p` here to derive the base of the path...?